tests/unit/resources/test_params.py

```diff
@@ -88,14 +88,25 @@ def test_service_action_params_constants(self):
         self.assertEqual(params['Param3'], True,
             'Parameter not set from boolean constant')
 
+    def test_service_action_params_input(self):
```

Comment:
I've gone ahead and updated this test to assert that no params get added automatically and user-supplied params get passed through unmodified.